src/controllers/legacycontrollersettings.h

```diff
@@ -0,0 +1,459 @@
+#pragma once
+
+#include <QDomElement>
+#include <QJSValue>
+#include <QLabel>
+#include <QLayout>
+#include <QList>
+#include <QLocale>
+#include <QObject>
+#include <QSizePolicy>
+#include <QSpinBox>
+#include <QString>
+#include <QWidget>
+#include <limits>
+#include <memory>
+#include <tuple>
```

Comment:
Some of these includes can be moved to the CPP file and here replaced by a forward declaration like 
`class QWidget;`
Please check the others as well.